src/web/browser/ga/ga.ts

```diff
@@ -189,3 +189,26 @@ export const trackNonClickInteraction = (actionName: string): void => {
         );
     }
 };
+
+export const trackPerformance = (
+    timingCategory: string,
+    timingVar: any,
+    timingLabel: string,
+): void => {
+    const { ga } = window;
+
+    if (!ga) {
+        return;
+    }
+
+    if (window.performance?.now) {
```

Comment:
is this to check that a performance test is in progress?

Reply:
it makes sure the [API](https://developer.mozilla.org/en-US/docs/Web/API/Performance) exists in the browser

Reply:
I feel like its a given these days https://caniuse.com/?search=performance but why not 🙂